code/modules/antagonists/heretic/knowledge/side_blade_rust.dm

```diff
@@ -61,7 +61,7 @@
 	route = PATH_SIDE
 
 /datum/heretic_knowledge/rifle_ammo
-	name = "Lionhunter Rifle Ammunition"
+	name = "Lionhunter Rifle Ammunition (free)"
```

Comment:
omittable detail in the name for brevity?

Reply:
i worry people wont realize its free though